web/src/color_picker_popover.ts

```diff
@@ -235,4 +235,14 @@ export function initialize(): void {
             toggle_color_picker_popover(target ?? this, stream_id);
         },
     );
+    $("body").on(
+        "click",
+        ".settings-field-label",
```

Comment:
This is too broad a class to listen for the click event on, and it also breaks the browsers's behavior of, for example, focusing on an input when the label is clicked. A better approach would be to add a new, unique class to the color-picker label, and listen for clicks on that.